CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.8 (2018-12-20)
+
+- Updated `@skyux/core` peer dependency to ensure proper modal host component removal [#20](https://github.com/blackbaud/skyux-modals/pull/20)
```

Comment:
Do you need this link if we're just talking about this PR? Didn't know if we had a pattern already established there...

Reply:
I think so. It is a weird one because I didn't feel like it was useful to make separate PRs for the change. However, down the road when people look back they will want to be able to see where the change came from and they won't have "this PR" to reference.